### ISSUE TITLE
showCellError SohoDataGridComponent 

### ISSUE THREAD

**Issue by Owaiii** [infor-cloud/m3-h5-sdk#206]
Hello, can you put the showCellError in SohoDataGridComponent ?

[design.infor.com >  data-grid  > component](https://design.infor.com/components/components/data-grid/enterprise/)

showCellError(row, cell, message, type)
Show the cell errors.

**Comment by tmcconechy** [comment]
Yes it appears missing as a type you can do (elem as any).showErrorCell as a workaround.
But for ids issues use our Jira project please https://inforwiki.atlassian.net/jira/software/c/projects/IDS/issues

**Comment by Owaiii** [comment]
Hello tmcconechy,

"(elem as any).showErrorCell" does not work... can you detail your solution? Maybe I'm not doing it very well.

And I don't have access to JIRA. thx
